src/utils/hooks/useGetReviewAssignment.tsx

```diff
@@ -15,8 +16,8 @@ interface UseGetReviewAssignmentProps {
 }
 
 const useGetReviewAssignment = ({ reviewerId, serialNumber }: UseGetReviewAssignmentProps) => {
-  const [assignment, setAssignment] = useState<AssignmentDetails | undefined>()
-  const [assignedSections, setAssignedSections] = useState<string[] | undefined>()
+  const [assignment, setAssignment] = useState<AssignmentDetails>()
+  const [sectionsAssigned, setSectionsAssigned] = useState<SectionDetails[]>()
```

Comment:
Would you mind adding a short "TO-DO" down around line 50 about handling the case where might have multiple assignments (for multiple levels) as we discussed?